packages/web/config/chain-infos.ts

```diff
@@ -1338,6 +1338,34 @@ const chainInfos = (
       features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
       explorerUrlToTx: "https://www.mintscan.io/evmos/txs/{txHash}",
     },
+    {
```

Comment:
can we move this to the bottom of the list please? Not sure why it's placed here.